src_python/mem.py

```diff
@@ -0,0 +1,36 @@
+def mem(num_of_nodes,num_of_edges,num_of_features,feature_data_type,edge_data_type):
+    
+    edge_mem = 2 * num_of_edges * (edge_data_type/8)
+    node_mem = num_of_nodes * num_of_features * (feature_data_type/8)
+
+    graph_size = (edge_mem + node_mem) / (1024*1024*1024)
+
+    return 3.6 * graph_size + 2
```

Comment:
Lets move 3.6 to meaningful constant

Reply:
done

Reply:
Base on the offline discussion we had, detailed comments have been added to explain how the parameters have been derived